docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -259,7 +327,10 @@ Standard TensorFlow\* operations:
 | Tile | No |
 | TopkV2 | No |
 | Transpose | No |
+| Unique | No |
 | Unpack | No |
+| Variable | No |
+| VariableV2 | No |
 | Where | No |
 | ZerosLike | No |
 
```

Comment:
Also, you probably missed TensorList* operations here. They are supported only when it is part of a sub-graph of the special form

Reply:
I don't want to claim that we support them. These ops are kind of "internal" ops.